trin-history/src/jsonrpc.rs

```diff
@@ -184,16 +184,34 @@ impl HistoryRequestHandler {
                         }
                     }
                 }
-                HistoryEndpoint::Offer(enr, content_key) => {
+                HistoryEndpoint::Offer(enr, content_key, content_value) => {
                     let enr = convert_enr(enr);
 
-                    let content_key: Vec<RawContentKey> = vec![content_key.as_ssz_bytes()];
-                    let response = match self.network.overlay.send_offer(content_key, enr).await {
-                        Ok(accept) => Ok(json!(AcceptInfo {
-                            content_keys: accept.content_keys,
-                        })),
-                        Err(msg) => Err(format!("SendOffer request timeout: {msg:?}")),
+                    let response = if let Some(content_value) = content_value {
+                        let mut content_item = vec![];
+                        content_value.encode(&mut content_item);
```

Comment:
Would it be preferable to have `content_value.into()` functionality rather than doing this little dance? I'm also not a huge fan of `content_value` and `content_item` naming convention. I think it's a result of #504? Imo it's not obvious from their name what they refer to / what is the difference between them. We've been using `content_value` pretty consistently throughout this codebase to refer to a `content_key`'s complement, which is the cause of my confusion. Moving forward I'd like to adopt one or the other ... unless I'm missing something here, which is totally possible